etl/transform.py

```diff
@@ -41,6 +51,14 @@ class ImageRecommendation:
         )
     )
 
+    filter_out: Column = (
+        F.when(
+            F.col("instance_of").isin(instance_of_to_filter_out),
```

Comment:
This is perfectly fine for now, but we should be mindful of performance bottlenecks with `Column.isin` in Spark 2.x.

For example: https://issues.apache.org/jira/browse/SPARK-33383

If the list grows (e.g. with placeholders), we might want to consider other strategies for managing the filter list (e.g. a broadcast join).